test/integration/consul-container/libs/cluster/helpers.go

```diff
@@ -49,15 +56,19 @@ func CreatingAcceptingClusterAndSetup(t *testing.T, numServer int, version strin
 		ToAgentConfig()
 	require.NoError(t, err)
 
-	t.Logf("dc1 client config: \n%s", clientConf.JSON)
+	t.Logf("%s client config: \n%s", clusterOpts.Datacenter, clientConf.JSON)
 
 	configs = append(configs, *clientConf)
 
 	cluster, err := New(configs)
 	require.NoError(t, err)
+	cluster.BuildContext = ctx
 
 	// Use the client agent as the HTTP endpoint since we will not rotate it
```

Comment:
Maybe instead of returning this, we should have a method on `cluster` that returns a client? That way, we could use it in `rotate_server_and_ca....go`.